Demo/Demo/SwiftUIComponents/CardVaultViews/CardVaultView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+
+struct CardVaultView: View {
+
+    @StateObject var baseViewModel = BaseViewModel()
+    @StateObject var cardVaultViewModel = CardVaultViewModel()
+
+    private let cardFormatter = CardFormatter()
+
+    // MARK: Views
+
+    var body: some View {
+        ScrollView {
+            ScrollViewReader { scrollView in
+                VStack(spacing: 16) {
+                    CreateSetupTokenView(baseViewModel: baseViewModel, cardVaultViewModel: cardVaultViewModel)
+                    if let setupTokenResponse = cardVaultViewModel.state.setupTokenResponse {
+                        SetupTokenResultView(setupTokenResponse: setupTokenResponse)
+                        let setupToken = setupTokenResponse.id
+                        UpdateSetupTokenView(baseViewModel: baseViewModel, cardVaultViewModel: cardVaultViewModel, setupToken: setupToken)
+                            .id("updateSetupTokenView")
```

Comment:
another noob q: Does the `id` modifier help prevent re-renders?

Reply:
It was strictly for the purpose of scrolling to those positions so the new views added are visible.

Reply:
Ah yeah I saw that shortly after I made this common in a web search. I wish there was a `scrollToBottom` method that would be legit, but if ID's work that's the wave 🌊 🏄 .

Reply:
Yeah, maybe you can do that with Geometry Reader.